components/blogger/actions/update-post/update-post.mjs

```diff
@@ -37,13 +37,19 @@ export default {
       ],
       optional: true,
     },
+    labels: {
+      type: "string[]",
+      label: "Labels",
+      description: "The list of labels this post is tagged with.",
+    },
   },
   async run({ $ }) {
     const {
       blog,
       post,
       title,
       content,
+      labels,
     } = this;
 
     let postData;
```

Comment:
I think we need to get the postData if labels doesn't exist.

`if (!content || !title) {`
should be:
`if (!content || !title || !(labels?.length > 0)) {`

Reply:
Just realized it has to exist because it's not optional. Could we make it optional in case someone wants to update just the title or content of a post?